web/src/main/java/no/nav/modiapersonoversikt/rest/dialog/SfDialogController.kt

```diff
@@ -0,0 +1,225 @@
+package no.nav.modiapersonoversikt.rest.dialog
+
+import no.nav.common.auth.subject.SubjectHandler
+import no.nav.modiapersonoversikt.infrastructure.naudit.Audit
+import no.nav.modiapersonoversikt.infrastructure.naudit.AuditIdentifier
+import no.nav.modiapersonoversikt.infrastructure.naudit.AuditResources
+import no.nav.modiapersonoversikt.infrastructure.tilgangskontroll.KjedeIdTilgangData
+import no.nav.modiapersonoversikt.infrastructure.tilgangskontroll.Policies
+import no.nav.modiapersonoversikt.infrastructure.tilgangskontroll.Tilgangskontroll
+import no.nav.modiapersonoversikt.legacy.api.domain.Oppgave
+import no.nav.modiapersonoversikt.legacy.api.domain.Temagruppe
+import no.nav.modiapersonoversikt.legacy.api.domain.sfhenvendelse.generated.models.HenvendelseDTO
+import no.nav.modiapersonoversikt.legacy.api.domain.sfhenvendelse.generated.models.SamtalereferatRequestDTO
+import no.nav.modiapersonoversikt.legacy.api.service.OppgaveBehandlingService
+import no.nav.modiapersonoversikt.service.sfhenvendelse.EksternBruker
+import no.nav.modiapersonoversikt.service.sfhenvendelse.SfHenvendelseService
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.http.HttpStatus
+import org.springframework.web.bind.annotation.*
+import org.springframework.web.server.ResponseStatusException
+
+@RestController
+@RequestMapping("/rest/sf-dialog")
+class SfDialogController @Autowired constructor(
```

Comment:
Forslag til fremtidig api for dialogene i modia. Krever en mye større endring i frontend en bruken av SfLegacy-controllerene. Men kan evt tittes på om tiden tillater det